inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/resample_opt.cl

```diff
@@ -53,11 +77,151 @@ KERNEL (resample_opt)(__global INPUT0_TYPE* input,
 #endif
 )
 {
+#ifdef SAMPLE_TYPE_CAFFE_INTERP
```

Comment:
Why don't you separate the function itself? It would be better in terms of readability

Reply:
done split